search-sentiments-map/public/scripts/map-script.js

```diff
@@ -66,17 +66,32 @@ function loadMapOutline() {
 
 /** Loads the country sentiment score from Datastore. */
 function loadCountryData() {
+  var dataVariableMax;
+  var countryMax = '';
+  var dataVariableMin; 
+  var countryMin = '';
```

Comment:
order them so that country are together and data are together.
Also add comments saying what they are.